CMakeLists.txt

```diff
@@ -397,6 +397,7 @@ add_library(Support STATIC
     LiteCore/Support/Error_android.cc
     LiteCore/Support/FilePath.cc
     LiteCore/Support/Logging_Stub.cc
+    LiteCore/Support/SecureRandomize.cc
```

Comment:
Without defining _CRYPTO_MBEDTLS in the Support library on non-Apple this will always result in a non-functioning `SecureRandomize`

Reply:
It gets added to `CMAKE_CXX_FLAGS` up on line 194; wouldn't that apply when building Support?

Reply:
That's only for Unix actually.  I don't know how it ended up like this but down on line 330 it defines it specifically for LiteCoreStatic on Windows (and also Android).